src/io/opendss.jl

```diff
@@ -1,6 +1,14 @@
 # OpenDSS parser
 
 
+"parses 'source_id' into Dict containing the 'type', 'name' and active 'phases' of the OpenDSS component"
+function parse_source_id(source_id::String)
+    component_type, component_name, active_phases = split(source_id, '.', limit=3)
+    nodes = [contains(active_phases, "$n") for n in 1:3]
```

Comment:
Instead of a list of booleans it may be easier to have a set of ints.  This will be easier to loop over and do `contains` checks.